projects/packages/forms/src/blocks/contact-form/util/block-fixtures.js

```diff
@@ -0,0 +1,463 @@
+import fs from 'fs';
```

Comment:
I've copied this file from `projects/plugins/jetpack/extensions/shared/test/block-fixtures.js`.
Also copied a few other files (inside `util/` older) necessary to build the blocks.
I'm wondering if there's a way we could avoid this duplication.

cc @ice9js

Reply:
We can rely on the `@automattic/jetpack-shared-extension-utils` package to host utilities and tools that we can use across multiple plugins / packages.

In practice,

1. In a first PR, you'd create the utility in the package here: `projects/js-packages/shared-extension-utils/`
2. In a follow-up PR, you'd switch the Jetpack plugin and the Forms package to start using the package.

Reply:
For this file, specifically, I ended up removing it and all the tests related files.
Since the tests for the blocks haven't been touched in a while and might need some rework, I thought I could keep this PR a bit cleaner and handle the tests in another PR.
Does that make sense?
For a few other common files, would it be possible to remove the duplicates in a follow-up PR?
It'd be great if we could merge this PR as soon as possible.